amethyst_assets/src/loader.rs

```diff
@@ -86,7 +86,7 @@ impl Loader {
     /// Registers an asset and inserts a context into the map.
     pub fn register<A, C>(&mut self, context: C)
         where A: Asset + 'static,
-              C: Context<Asset=A> + 'static,
+              C: Context<Asset=A> + Send + Sync + 'static,
```

Comment:
I don't want to overflow trait with bounds.
But if we not going to use it without `Loader` then it's OK I think